packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -151,6 +149,18 @@ const Redirect = ({
             }
             break;
         }
+        case 'manage_account': {
+            WS.wait('get_account_status').then(() => {
+                if (!currency) return openRealAccountSignup('set_currency');
```

Comment:
Question: Do we need this line since in this case the user already has a real account ?

Reply:
Nope. Will remove it.